geniza/corpus/tei_transcriptions.py

```diff
@@ -24,6 +24,10 @@ class GenizaTei(teimap.Tei):
     labels = xmlmap.NodeListField(
         "tei:text/tei:body/tei:div/tei:label", GenizaTeiLine
     )  # not really a line...
+    # source description sometimes contains reference to scholarship record
+    source = xmlmap.NodeListField(
+        "tei:teiHeader//tei:sourceDesc/tei:msDesc/tei:msContents/tei:p", GenizaTeiLine
```

Comment:
Wow! Does this then become a TEI `<source>` element?

Reply:
I'm trying to keep the tei handling as simple as I can, because I just want to easily access the parts of the content that I care about and don't need to manipulate the xml. (I don't remember why I used the `GenizaTeiLine` class, could probably be use more generic...) 

Some of the TEI files describe the source of the transcription in text inside `<p>` tags nested under the `<sourceDesc>` — this mapping gives me a list of lines that I can convert to string so I can easily check if e.g. "Goitein, typed texts" is referenced.

This mapping lets me access those paragraphs as an array on the tei object, so that for example I can check if Goitein and typed texts occur within the text content of `tei.source[0]`  (you'll see more on a subsequent pr!)

Reply:
I see! That makes a lot of sense, thanks.